backend/app/rag/llms/provider.py

```diff
@@ -8,6 +8,7 @@ class LLMProvider(str, enum.Enum):
     OPENAI = "openai"
     GEMINI = "gemini"
     ANTHROPIC_VERTEX = "anthropic_vertex"
```

Comment:
Keep the LLMProvider.ANTHROPIC_VERTEX enumeration to avoid errors for users who are already using it.

Reply:
It is probably only us who are using ANTHROPIC_VERTEX, so I suggest to delete it in the code. we(tidb.ai) can re-configure it in production environment.